newdle/api.py

```diff
@@ -356,6 +358,29 @@ def update_participant(args, code, participant_code):
     for key, value in args.items():
         setattr(participant, key, value)
     db.session.commit()
+    formatted_answers = []
+    for date, state in participant._answers.items():
```

Comment:
You can use a list comprehension instead

Reply:
Interesting feature 🤔